registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -682,6 +685,7 @@
     "sha256:3a286db827d07a6f99c8051ddefa08dc8539a48b90113b1e38f416c3dd977673": ["v1.11.0-rc.1"]
     "sha256:33e69a37fbc131dadd965e81b3219ba739a24308d27a4e8c52f2136867b8b82e": ["v1.11.1"]
     "sha256:91bd4deea6cf2c8bec93d7ea8c10836df2e93ff27acf6d7a0f93916d32e88557": ["v1.11.2"]
+    "sha256:760396dd21fa013fe8d09fbd76cc55c5682a18fc39eebc1d63399e4fa8d1cc2f": ["v1.11.3"]
```

Comment:
SHA `760396dd21fa` matches kubeadm-control-plane-controller v1.11.3 in the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-control-plane-controller@sha256:760396dd21fa013fe8d09fbd76cc55c5682a18fc39eebc1d63399e4fa8d1cc2f/details).